src/Acuminator/Acuminator.Analyzers/Analyzers/DAC/PXGraphUsageInDacAnalyzer.cs

```diff
@@ -45,7 +45,18 @@ public GraphUsageInDacWalker(SyntaxNodeAnalysisContext syntaxContext, PXContext
                 _pxContext = pxContext;
             }
 
-            public override void VisitMethodDeclaration(MethodDeclarationSyntax node)
+	        public override void VisitClassDeclaration(ClassDeclarationSyntax node)
```

Comment:
@dropsonic we should visit attribute list nodes and skip analysis in them. We have an issue report about that